v1.0-known-limitations.md

```diff
@@ -0,0 +1,38 @@
+---
+title: v1.0 Known Limitations
+summary:
+toc: false
+---
+
+This page identifies limitations we've identified in the v1.0 release and plan to investigate and possibly resolve in a future release.
+
+<div id="toc"></div>
+
+## `INSERT ON CONFLICT` vs. `UPSERT`
+
+When inserting/updating all columns of a table, and the table has no secondary indexes, it's recommended to use an `UPSERT` statement instead of the equivalent `INSERT ON CONFLICT` statement. Wherease the `INSERT ON CONFLICT` always has to read to determine the necessary writes, the `UPSERT` statement writes without first reading and so will be faster.
+
+This limitations may be particularly relevant if you are using a simple SQL table of two columns to [simulate direct KV access](frequently-asked-questions.html#can-i-use-cockroachdb-as-a-key-value-store). In that case, be sure to use the `UPSERT` statement.
+
+## `unique_rowid()` values in Javascript
```

Comment:
@petermattis, the following is based on https://github.com/cockroachdb/cockroach/issues/15721. Please let me know if we need more details, and if we have a specific suggested workaround.